test/node/rest-api/request/body/body-form-data.node.test.ts

```diff
@@ -9,6 +9,20 @@ const server = setupServer(
     const formData = await request.formData()
     return HttpResponse.json(Array.from(formData.entries()))
   }),
+  http.post('http://localhost/file', async ({ request }) => {
+    const formData = await request.formData()
+    const file = formData.get('file') as File | null
+
+    if (!file) {
+      throw HttpResponse.text('Missing file', { status: 400 })
+    }
+
+    return HttpResponse.json({
+      name: file.name,
```

Comment:
This verifies that the Blob sent from the request is properly available in the response resolver.